docs/upgrades/locks/ez_35.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: ez_35
+---
+
+## Description
+
+Halperyon Systems' EZ 35. A step up from the 21, but not much.
+
+### Stats
+
+ez_35 has no additional stats. The upgrade only spawns at 'noob' rarity.
+
+## Behavior
+
+When encountered, the lock expects a string as its first parameter, and a digit as its second. Each parameter will return corresponding errors if an incorrect data type is input.
+
+## Solving
+
+<details>
+  <summary>Spoilers for ez_35's solutions.</summary>
+  The first parameter's possible strings are, "open", "unlock", and "release". The
+  second parameter is a single digit, input as a number, ranging from 0 to 9.
```

Comment:
Add some newlines here to break the paragraph correctly, which fixes the summary thing

```suggestion
  <summary>Spoilers for ez_35's solutions.</summary>
  
  The first parameter's possible strings are, "open", "unlock", and "release". The
  second parameter is a single digit, input as a number, ranging from 0 to 9.
  
```